src/Config.php

```diff
@@ -20,10 +20,12 @@
 
 class Config
 {
+    protected const IGNORE_TAG = "php-cs-fixer:ignore-file";
```

Comment:
In readme you mentioned adding `php-cs-fixer-ignore-file` tag and here you're implementing `php-cs-fixer:ignore-file`. Please unify this.

Reply:
Also: what happen to this file when someone would enable ignoring files?

Reply:
unified

Reply:
turning on the ignored file check is to check for exclusions, and if you haven't turned it on, the fixer will simply not check them - it's to improve performance when you don't use it